examples/consumeBatch.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+use Amp\Future;
+use Thesis\Amqp\Client;
+use Thesis\Amqp\Config;
+use Thesis\Amqp\ConsumeBatch;
+use Thesis\Amqp\DeliveryMessage;
+use Thesis\Amqp\Message;
+use Thesis\Amqp\PublishMessage;
+use function Amp\async;
+use function Amp\trapSignal;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+$client = new Client(Config::fromURI('amqp://thesis:secret@localhost:5673'));
+$channel = $client->channel();
+
+$queue = $channel->queueDeclare(autoDelete: true);
+
+$channel->confirmSelect();
+
+$channel
+    ->publishBatch(array_map(
+        static fn(int $number): PublishMessage => new PublishMessage(new Message("{$number}"), routingKey: $queue->name),
+        range(1, 8),
+    ))
+    ->awaitAll();
+
+$consumerTag = $channel->consumeBatch(
+    static function (ConsumeBatch $batch): void {
+        dump(array_map(static fn(DeliveryMessage $delivery): string => $delivery->message->body, $batch->deliveries));
```

Comment:
```suggestion
        foreach ($batch->deliveries as $delivery) {
            dump($delivery->message->body);
        }
```

This is much easier to get when reading.